capplets/windows/mate-window-properties.c

```diff
@@ -315,68 +325,61 @@ main (int argc, char **argv)
         return 1;
     }
 
-    marco_settings = g_settings_new (MARCO_SCHEMA);
-
     builder = gtk_builder_new ();
     if (gtk_builder_add_from_resource (builder, "/org/mate/mcc/windows/window-properties.ui", &error) == 0) {
         g_warning ("Could not load UI: %s", error->message);
         g_error_free (error);
-        g_object_unref (marco_settings);
         g_object_unref (builder);
         return -1;
     }
```

Comment:
You can use [gtk_builder_new_from_resource](https://developer.gnome.org/gtk3/stable/GtkBuilder.html#gtk-builder-new-from-resource) instead of gtk_builder_new + gtk_builder_add_from_resource, as gtk_builder_new_from_resource interrupts the execution if it doesn't find the resource.